src/index.html

```diff
@@ -5,9 +5,54 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card" data-qa="card">
+      <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="imac"
+      >
+
+      <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+      <div class="card__code">
+        Product code: 195434
+      </div>
+
+
+      <div class="card__reviews">
+        <div class="stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
```

Comment:
reuse all tags and styles from stars task
```suggestion
        <div class="stars stars--4">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </div>
```

Reply:
`
![image](https://github.com/mate-academy/layout_product-cards/assets/91217030/f6cb70a2-4f43-4fa2-ab5a-eaae2a9e83ef)
`why i need this ?

Reply:
![image](https://github.com/mate-academy/layout_product-cards/assets/91217030/7c1a9826-2cca-493c-be57-17d280c172d3)
why i need it ?

Reply:
becouse u should have ability to change count of the stars directly from html

Reply:
<img width="280" alt="Screenshot 2023-09-13 at 13 52 01" src="https://github.com/mate-academy/layout_product-cards/assets/135232001/e64439e5-fc50-49e6-972a-11d29778f240">
Not fixed